pkg/resource/plugin/langruntime_plugin.go

```diff
@@ -53,7 +53,15 @@ func NewLanguageRuntime(host Host, ctx *Context, runtime string) (LanguageRuntim
 		})
 	}
 
-	plug, err := newPlugin(ctx, path, runtime, []string{host.ServerAddr()})
+	var args []string
+	for k, v := range options {
+		if v {
+			args = append(args, "-"+k)
```

Comment:
This means that anything in the `options` section of `Pulumi.yaml` will get sent as command-line arguments, right? Even things like `logtostderr`?

Reply:
> This means that anything in the options section of Pulumi.yaml will get sent as command-line arguments, right? Even things like logtostderr?

Correct.  (Note that options which appear on Pulumi's command line do not get flowed automatically).  I suppose you could use this as a poorman's logflow.

Note I am changing this slightly such that every value is passed, either as `-optionName=true` or `-optionName=false`, to support typescript by default and require you to explicitly opt out.